pkgs/tools/compression/lzip/default.nix

```diff
@@ -17,6 +17,7 @@ stdenv.mkDerivation rec {
 
   patches = lib.optionals stdenv.hostPlatform.isMinGW [
     ./mingw-install-exe-file.patch
+    ./mingw-mkdir-one-argument.patch
```

Comment:
Suggestion: Add a comment above the patch that says `Can be removed on next release:`. (IIUC)

Reply:
fixed